opencensus-shim/src/integrationTest/java/io/opentelemetry/opencensusshim/IntegrationTest.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.opencensusshim;
+
+import io.opencensus.trace.AttributeValue;
+import io.opencensus.trace.Tracing;
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.api.trace.SpanKind;
+import io.opentelemetry.api.trace.Tracer;
+import io.opentelemetry.context.Scope;
+import org.junit.jupiter.api.Test;
+
+// todo remove if/once the underlying issue is agreed upon
+public class IntegrationTest {
+
+  @Test
+  void test() {
```

Comment:
What is this testing? The class has a nondescript name, the method has a nondescript name, and there aren't any assertions.